GetServiceByRevision/handler.ts

```diff
@@ -128,7 +132,11 @@ export function GetServiceByRevisionHandler(
   serviceModel: ServiceModel
 ): IGetServiceByRevisionHandler {
   return async (serviceId, version) =>
-    getServiceByRevisionTask(serviceModel, serviceId, version)
+    getServiceByRevisionTask(
+      serviceModel,
+      serviceId,
+      Number(version) as NonNegativeInteger
```

Comment:
you don't need this if you use `IntegerFromString` in the middleware

Reply:
Fixed in [7849fcb](https://github.com/pagopa/io-functions-assets/pull/2/commits/7849fcb0ecf2401a816bd7fe2f2f3265d5aa00a9)